### ISSUE TITLE
Missing Conditioning Component Statistics Results

### ISSUE THREAD

**Issue by hbarton-atsec** [usnistgov/ESV-Server#45]
The ESV server seems to be missing the statistical results for the Conditioning Component. Here's an example from ESV demo server with testID 4916.

[
  {
    "esvVersion": "1.0"
  },
  {
    "id": 11128,
    "status": "RunSuccessful",
    "results": {
      "sha256": "73194eba53646f352139144a7c7e7aba74c5e844b9fc57d639bf67f46faae25f",
      "commandline": "./ea_conditioning -n 4096 64 64 64 -q -c non-iid -i /app/dfc_d53dba89-8deb-ee11-82c7-122bb4a6746b.bin -o /app/11128-conditioned-output.json",
      "filename": "/app/dfc_d53dba89-8deb-ee11-82c7-122bb4a6746b.bin",
      "errorLevel": 0,
      "errorMessage": null,
      "toolVersion": "1.1.7",
      "testCases": [
        {
          "h_in": 64.0,
          "h_out": 57.462893069088814,
          "h_p": 0.89785770420451271,
          "n_in": 4096.0,
          "n_out": 64.0,
          "nw": 64.0,
          "testCaseDesc": "Overall"
        }
      ]
    },
    "sha256": "73194eba53646f352139144a7c7e7aba74c5e844b9fc57d639bf67f46faae25f"
  }
]

**Comment by celic** [comment]
We run the file to determine H_p, the per-bit entropy rate. We just don't communicate the full non-iid test results back out to the user. I'll see if this can be fixed in a future release.

**Comment by hbarton-atsec** [comment]
Yes if this could be added in the future that would be great since this info needs to be included in the EAR. Thanks!

**Comment by hbarton-atsec** [comment]
Yup this is the approach we currently take. Thanks for the clarification.

**Comment by celic** [comment]
This should be available now on the server.
